Feature/wam hackaton ci and google chrome csv import

gitcoindev opened the pull request:
Hi, this is my submission for WAM hackathon, I described the pull request at https://hackmd.io/@en6I0FFLT6iJvOcdkCuOXg/SkgEOqlGc . Thank you for reviewing it!

cryptomental commented on README.md:
```diff
@@ -1,5 +1,7 @@
 # FairPass
 
+[![Build](https://github.com/gitcoindev/FairPass/workflows/Build/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ABuild) [![Test](https://github.com/gitcoindev/FairPass/workflows/Test/badge.svg)](https://github.com/gitcoindev/FairPass/actions?query=workflow%3ATest) [![Static code analysis](https://github.com/gitcoindev/FairPass/workflows/Static%20code%20analysis/badge.s ...
```
I noticed the badges point to my fork, after merging the github.com/gitcoindev path will change to github.com/fairDataSociety, apart from that no changes will be needed.

gitcoindev commented:
Closing the pull request as won't merge. There was no communication and will to review for the last 10 days.

crtahlin commented:
Hi @gitcoindev 

Thank you for your participation at WAM and your work you did on this issue. 

Although it didn't meet the complete requirement list, the parts you did would be very useful and supportive for the FairOS  Password manager workflow. 

If you would be interested, we can also continue on the FDS Discord (sent you a friend request, I hope I have the right user).

gitcoindev commented:
Hi @crtahlin , @asabya I am back. Please let me know which parts of the pull request to remove or rework. Let's work on merging this ! -)

asabya commented:
Hello @gitcoindev, The changes in the PR can be easily merged. Just wanted to ask one thing, is there any specific reason that you didn't use [matrix](https://docs.github.com/en/actions/using-jobs/using-a-build-matrix-for-your-jobs) for the build process on different operating system?

gitcoindev commented:
Hi @asabya, there was no specific reason. I can try to refactor the build process to use matrix, will let you know when it's completed. Should be ready today, tomorrow at the latest.

gitcoindev commented:
Hi @asabya I refactored the build process to use GitHub matrix and I also added release configuration with GitHub matrix support. An example is available at https://github.com/gitcoindev/FairPass/releases/tag/v0.0.8 and the logs at https://github.com/gitcoindev/FairPass/actions/runs/2196043913 .

tfius commented:
> LGTM. @tfius @crtahlin should we merge this?

yes
